net_utils.js

```diff
@@ -421,3 +428,32 @@ exports.get_ipany_re = function (prefix, suffix, modifier) {
         modifier
     );
 };
+
+exports.ipv6_reverse = function(ipv6){
+    ipv6 = new IPv6.Address(ipv6);
```

Comment:
toNormalizedString().split(':').reverse().join('.') + '.ip6.arpa' 

Should provide the same I think.